modules/service/src/test/scala/lucuma/odb/graphql/feature/reference.scala

```diff
@@ -267,154 +252,22 @@ class reference extends OdbSuite {
     )
   }
 
-  val ref2010A1 = "G-2010A-0001".proposalReference
-
-  // TODO: proposal status mutations
-  // I think we want to remove semester as an independently settable property.
-  // It should instead be required on proposal submission.
-  test("TEMP: set semester on create, then submit") {
-    val res = for {
-      pid <- createProgramWithSemester("2010A")
-      _   <- addProposal(pi, pid)
-      ref <- submitProposal(pi, pid, none) // no semester
-    } yield ref
-
-    assertIO(res, ref2010A1)
-  }
-
-  // TODO: proposal status mutations
-  // I think we want to remove semester as an independently settable property.
-  test("TEMP: cannot unset semester after submit") {
```

Comment:
Is it now not possible to unset the semester via the API after a science program reference has been established?

Reply:
As discussed in the comment in `ProgramService`, the ability to change/unset the semester has been left in `updatePrograms` until we get a way to set the Call for Proposal, which will also set the semester. We could make it so the API does not allow unsetting the semester for now to avoid the problem until we have another method of setting the semester.

Reply:
Right, I think that seems reasonable.

Reply:
OK. I'll do that.

Reply:
Done